<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnipSysTray</class>
 <widget class="QWidget" name="SnipSysTray">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnipSysTray</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hot Key:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="ToolLineEdt" name="hotKeyEdt"/>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Run On System Startup:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Quick Save Dir:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="ToolLineEdt" name="saveDirEdt"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolLineEdt</class>
   <extends>QLineEdit</extends>
   <header>toollineedt.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
